### ISSUE TITLE
ERROR Provision transaction already in progress

### ISSUE THREAD

**Issue by telekid** [keybase/client#5742]
I just updated the keybase app on my mac, haven't used it in a while. I had to login; during that process, I was asked to run `keybase device add`. That resulted in `ERROR Operation timed out`. I then quit the keybase app and tried to login again through the app. Keybase threw `ERROR Provision transaction already in progress`. Running `keybase login` resulted in the same error.

my log id: 6eafbb90fff3b4a223311a1c

**Comment by maxtaco** [comment]
Did you issue `keybase device add` on the same machine, or a different machine?

CC: @patrickxb

**Comment by telekid** [comment]
I ran all of the above commands on the same machine. Additionally, I believe the machine that I selected in the question following the username prompt is the same machine that I ran the above commands on. (I believe that question was something along the lines of "what device would you like to link to," but I didn't read it too thoroughly. Perhaps you can remind me what that question is?)

**Comment by maxtaco** [comment]
Yeah, you were dropped into a flow where one machine was trying to provision another.

Cc: @malgorithms  on the confusion here, which might be relatively common

**Comment by telekid** [comment]
IIRC, it wasn't clear what to do because the device that I was trying to provision was already on the device list. I chose the device I was already using as the "from" device, which felt odd, but I don't recall a different option that made sense.

Once I can login again, I'll give it another shot and try to further clarify where my confusion came from. Until then, I'm stuck at `Provision transaction already in progress.`

**Comment by telekid** [comment]
I successfully logged in after today's update. However, the account link screen could use more context:

![confusing account link screen](https://hz.keybase.pub/confusing-link-screen.png)

For example, it isn't immediately clear what connecting with a device does. What happens if you connect from the same device that you are currently using? Why is your paper key listed as a device? Etc.

Otherwise, nice job on the chat app! Very cool and easy to use.
